crates/cast/tests/data/scripts/hello_world/hello_world.cairo

```diff
@@ -0,0 +1,65 @@
+use array::ArrayTrait;
+use array::SpanTrait;
+use traits::Into;
+use traits::TryInto;
+use serde::Serde;
+use traits::Drop;
+
+use debug::PrintTrait;
+
+use starknet::{
+    testing::cheatcode, ClassHash, ContractAddress, ClassHashIntoFelt252,
+    ContractAddressIntoFelt252, Felt252TryIntoClassHash, Felt252TryIntoContractAddress,
+    contract_address_const
+};
+
+#[derive(Drop, Clone)]
+struct CallResult {
+    data: Array::<felt252>,
+}
+
+fn call(
+    contract_address: ContractAddress, function_name: felt252, calldata: Array::<felt252>
+) -> CallResult {
+    let contract_address_felt: felt252 = contract_address.into();
+    let mut inputs = array![contract_address_felt, function_name];
+
+    let calldata_len = calldata.len();
+    inputs.append(calldata_len.into());
+
+    let mut i = 0;
+    loop {
+        if i == calldata_len {
+            break;
+        }
+        inputs.append(*calldata[i]);
+        i += 1;
+    };
+
+    let buf = cheatcode::<'call'>(inputs.span());
```

Comment:
do you mind explaining to me how will this look like after call is moved from separate module? 

ideally we would not have to call `cheatcode::<'call'>` but rather (at least) hide it behind some abstraction like just `let buf = call(...)`, but I'm still new to all those cheatcodes logic so perhaps it's just not possible and I just don't know something.

Reply:
It will work like this: https://github.com/foundry-rs/starknet-foundry/pull/1111/files#diff-6e5eba2cddaafbe74f218b4df832f8f280665ccfe3352ff14a1362f188fb575eR1-R13

Reply:
Great, thanks! Will it be problematic to have multiple scarb packages in one repo? Maybe we should think about moving this elsewhere?

Reply:
As per @maciektr, it shouldn't be a problem.

Reply:
Scarb searches for a package that matches the name provided in Scarb.toml

Reply:
perfect, thanks guys!